secant/Utils/Zatoshi.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Zatoshi.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 26.05.2022.
+//
+
+import Foundation
+
+typealias ZEC = NSDecimalNumber
+
+// MARK: - Zatoshi
+
+struct Zatoshi {
+    @Clamped(-(100_000_000 * 21_000_000)...(100_000_000 * 21_000_000))
```

Comment:
Could we put some of these values into named constants for clarity?  Then they could be re-used in the instances they appear below too.

Reply:
moved to Constants